src/main/java/org/takes/facets/auth/social/PsTwitter.java

```diff
@@ -65,26 +72,60 @@
     private final transient String key;
 
     /**
+     * Request for fetching app token.
+     */
+    private final transient com.jcabi.http.Request trequest;
+
+    /**
+     * Request for verifying user credentials.
+     */
+    private final transient com.jcabi.http.Request vcrequest;
```

Comment:
@adko-pl don't see much reason for this variable since it's url is re-created with the access token all the time.